src/background/Program.ts

```diff
@@ -2083,11 +2087,23 @@ export default class BackgroundProgram {
       raw: rawOptions,
       errors: [],
       mac,
+      chromiumVariant,
     };
 
     log.level = options.logLevel;
   }
 
+  async getChromiumVariant(): Promise<string> {
```

Comment:
```suggestion
  async getChromiumVariant(): Promise<"chrome" | "vivaldi"> {
```

This change makes it so that TypeScript yells at us if we try to use any other string than those. Could you use this in the message sent to the worker as well?